core/domain/user_jobs_continuous.py

```diff
@@ -204,8 +204,11 @@ def map(item):
         for exp_model in tracked_exp_models_for_feedback:
             threads = feedback_services.get_all_threads(exp_model.id, False)
             for thread in threads:
-                if thread['full_thread_id'] not in feedback_thread_ids_list:
-                    feedback_thread_ids_list.append(thread['full_thread_id'])
+                full_thread_id = (feedback_models.FeedbackThreadModel
```

Comment:
FYI, we generally do:

    full_thread_id = (
        feedback_models.FeedbackThreadModel.generate_full_thread_id(
            exp_model.id, thread['thread_id']))

This gives more space on the right.